dev/ci/README.md

```diff
@@ -137,3 +137,26 @@ Currently, available artifacts are:
 
 As an exception to the above, jobs testing that compilation triggers
 no OCaml warnings build Coq in parallel with other tests.
+
```

Comment:
You need to update the above section to instruct users to enable their private registry as otherwise Gitlab will fail now.